core/src/main/java/org/apache/carbondata/core/metadata/schema/table/TableInfo.java

```diff
@@ -77,6 +77,19 @@
    */
   private String tablePath;
 
+  /**
+   * The boolean field which points if the data written for UnManaged Table
+   * or Managed Table. The difference between managed and unManaged table is
+   * unManaged Table will not contain any Metadata folder and subsequently
+   * no TableStatus or Schema files.
+   */
+  private boolean isUnManagedTable;
+
+  /**
+   * Unique ID
+   */
+  private long UUID;
```

Comment:
UID cannot be in Tablelevel datastructure, as it is unique for one load. Move it to LoadModel

Reply:
removed